docs/interactions/Message_Components.md

```diff
@@ -201,3 +203,213 @@ When a user clicks on a button, your app will receive an [interaction](#DOCS_INT
     "application_id": "290926444748734465"
 }
 ```
+
+## Select Menus
+
+Select menus are another interactive component that renders on messages. On desktop, clicking on a select menu opens a dropdown-style UI; on mobile, tapping a select menu opens up a half-sheet with the options.
+
+![A select menu open on desktop](desktop-select.png)
+![A select menu open on mobile](mobile-select.png)
+
+Select menus support single-select and multi-select behavior, meaning you can prompt a user to choose just one item from a list, or multiple. When a user finishes making their choice by clicking out of the dropdown or closing the half-sheet, your app will receive an [interaction](#DOCS_INTERACTIONS_SLASH_COMMANDS/interaction-object-interaction-structure).
+
+- Select menus must be sent inside an Action Row
+- An Action Row can contain only one select menu
+- An Action Row containing a select menu cannot also contain buttons
+
+###### Select Menu Example
+
+```json
+// This is a message
+{
+    "content": "Mason is looking for new arena partners. What classes do you play?",
+    "components": [
+        {
+            "type": 1,
+            "components": [
+                {
+                    "type": 3,
+                    "custom_id": "class_select_1",
+                    "options":[
+                        {
+                            "label": "Rogue",
+                            "value": "rogue",
+                            "description": "Sneak n stab",
+                            "emoji": {
+                                "name": "rogue",
+                                "id": "625891304148303894"
+                            }
+                        },
+                        {
+                            "label": "Mage",
+                            "value": "mage",
+                            "description": "Turn \"em into a sheep",
+                            "emoji": {
+                                "name": "mage",
+                                "id": "625891304081063986"
+                            }
+                        },
+                        {
+                            "label": "Priest",
+                            "value": "priest",
+                            "description": "You get heals when I\"m done doing damage",
```

Comment:
```suggestion
                            "description": "You get heals when I'm done doing damage",
```

Reply:
For some reason I do not see these changes in my local editor, which is very odd.